### PR TITLE
try apply yaml in best efforts

### PR DESCRIPTION
**Pull request by xuzhenglun**
**What this PR does / why we need it**:
1. When I apply CR with it's CRD, an error like `resource <Kind> is not found` will return. 
2. In fact, `kubectl` will try its best efforts to apply YAMLs. In the progress, maybe CRDs have not been established and an error could return. But in the next retry, all YAMLs can be applied successfully.

ref: https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/apply/apply.go#L438

**Which issue(s) this PR fixes**:

Fixes #
**Special notes for your reviewer**:

**Additional documentation**:

### CODE REVIEW

File: pkg/patterns/declarative/pkg/applier/direct.go [hunk 1]
```diff
@@ -79,7 +85,7 @@ func (d *DirectApplier) Apply(ctx context.Context, opt ApplierOptions) error {
 	applyCmd := apply.NewCmdApply(baseName, f, ioStreams)
 	applyOpts, err := applyFlags.ToOptions(applyCmd, baseName, nil)
 	if err != nil {
-		return fmt.Errorf("error getting apply options: %w", err)
+		return utilerrors.NewAggregate(append(errs, fmt.Errorf("error from SetNamespace: %w", err)))
```
Comment (atoato88) [inline pkg/patterns/declarative/pkg/applier/direct.go:88]: Is message of `error from SetNamespace` correct?

File: pkg/patterns/declarative/pkg/applier/direct.go [hunk 2]
```diff
@@ -106,7 +112,8 @@ func (d *DirectApplier) Apply(ctx context.Context, opt ApplierOptions) error {
 		IOStreams: ioStreams,
 	}
 
-	return applyOpts.Run()
+	err = applyOpts.Run()
+	return utilerrors.NewAggregate(append(errs, fmt.Errorf("error from SetNamespace: %w", err)))
```
Comment (atoato88) [inline pkg/patterns/declarative/pkg/applier/direct.go:116]: ditto.

**Review by atoato88 — COMMENTED** [review]
Thank you to create PR.
It looks almost good but left comment.
Please check those.

File: pkg/patterns/declarative/pkg/applier/direct.go (hunk 2 above)
Comment (xuzhenglun) [reply pkg/patterns/declarative/pkg/applier/direct.go]: ho, I just corrected it.

**Comment by xuzhenglun** [comment]
@[atoato88](https://github.com/atoato88) Is anything else I need to do?

**Pull request merged**